twilio/twilio.ts

```diff
@@ -0,0 +1,11 @@
+const accountSid = process.env.TWILIO_SID
+const authToken = process.env.TWILIO_AUTH_TOKEN
+const client = require('twilio')(accountSid, authToken)
+
+export async function sendSMS(phoneNumber: string, message: string) {
+  client.messages.create({
+    body: message,
+    from: '+16066579480',
```

Comment:
should we make this a secret? 
```suggestion
    from: process.env.CLIENT_PHONE_NUMBER,
    
```

Reply:
I think since it's public info and won't change it doesn't make sense to make it a secret. 

making it a const makes more sense imo